FBAllocationTrackerTests/FBAllocationTrackerGenerationsTests.mm

```diff
@@ -163,4 +164,15 @@ - (void)testThatManagerWillFetchInstancesOfGivenClassFromGivenGeneration
   }
 }
 
+- (void)testThatNSZobmieClassDetectionWorks {
+  if (!fb_isNSZombieEnabled()) { return; }
+  
+  __unsafe_unretained id deallocatedObject = nil;
+  @autoreleasepool {
+    id testObject = [NSObject new];
+    deallocatedObject = testObject;
+  }
+  
+  XCTAssertTrue(fb_isZombieObject(deallocatedObject));
+}
```

Comment:
One small style nit, can we add empty line after the function, before `@end` clause?